### PR TITLE
REL-4202 Show Ghost targets on the TPE

### PR DESCRIPTION
**Pull request by cquiroz**
I'm not totally sure if these are correct. Some samples:

* Standard Single
Single normal zoom

* Standard Dual
Dual High zoom

* Standard + Sky
SkySR

* HR Single
HRIFU

### CODE REVIEW

File: bundle/jsky.app.ot/src/main/scala/jsky/app/ot/tpe/feat/TpeGhostIFUFeature.scala [hunk 1]
```diff
@@ -494,8 +483,8 @@ object TpeGhostIfuFeature {
 
 
   // Color for IFU limts.
-  private val IfuFovColor: Color = Color.RED
-  private val PatrolRangeColor: Color = OtColor.SALMON
+  val IfuFovColor: Color = Color.RED
```
Comment (swalker2m) [inline bundle/jsky.app.ot/src/main/scala/jsky/app/ot/tpe/feat/TpeGhostIFUFeature.scala:486]: I think it was requested this way but the red labels make me think an error is being indicated.

**Review by swalker2m — APPROVED** [review]
Looks good to me.

**Comment by bryanmiller** [comment]
This looks pretty good. I'm concerned that people will associate the red color with an error. How does it look to use the same pink color as for the cross-circle markers in two-target mode?

**Comment by cquiroz** [comment]
Maybe it is better to match the user target with yellow
image

**Comment by swalker2m** [comment]
That looks good to me

**Pull request merged**